content/ch-quantum-hardware/measuring-quantum-volume.ipynb

```diff
@@ -47,7 +47,7 @@
     "from qiskit.providers.aer.noise.errors.standard_errors import depolarizing_error, thermal_relaxation_error\n",
     "\n",
     "#Import the qv function\n",
-    "import qiskit.ignis.verification.quantum_volume as qv"
+    "import qiskit_experiments.library.quantum_volume as qv"
```

Comment:
```suggestion
    "import qiskit.ignis.verification.quantum_volume as qv"
```

Unfortunately this isn't a drop-in replacement as the page relies too heavily on old ignis functionality. I think we leave this as-is; the new textbook will soon have an updated version that uses `qiskit_experiments` (see commit https://github.com/Qiskit/platypus/pull/1505/commits/846a7c5bf7304cf06432577d044a34f7bca9a5bd of PR https://github.com/Qiskit/platypus/pull/1505).